frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -24,6 +24,9 @@ import { capitalizeFirstLetter, compactNumber } from 'lib/utils'
 import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
 import { ProductPricingModal } from './ProductPricingModal'
 import { PlanComparisonModal } from './PlanComparison'
+import { UnsubscribeSurveyModal } from './UnsubscribeSurveyModal'
+
+const UNSUBSCRIBE_SURVEY_ID = '018b6e13-590c-0000-decb-c727a2b3f462'
```

Comment:
since this is all coming via API and custom, you _could_, if you wanted to, add the product type property to the survey response, and then filter by that in the survey results page when you want to look at a specific product only.

Reply:
We have decided to add another question to the survey that will prompt for product type.

Reply:
Wait no not prompt, it'll just have it as an automatically filled field in the survey. We know what the product type is when they trigger it.